Code/Editor/Plugins/EditorAssetImporter/AssetImporterWindow.cpp

```diff
@@ -597,9 +600,11 @@ void AssetImporterWindow::SetTitle(const char* filePath)
 
 void AssetImporterWindow::UpdateSceneDisplay(const AZStd::shared_ptr<AZ::SceneAPI::Containers::Scene> scene) const
 {
-    AZ::IO::FixedMaxPath projectPath = AZ::Utils::GetProjectPath();
-    AZ::IO::FixedMaxPath relativeSourcePath = AZ::IO::PathView(m_fullSourcePath).LexicallyProximate(projectPath);
-    auto sceneHeaderText = QString::fromUtf8(relativeSourcePath.c_str(), static_cast<int>(relativeSourcePath.Native().size()));
+    QString sceneHeaderText;
+    if(scene.get())
+    {
+        sceneHeaderText = QString::fromUtf8(scene->GetManifestFilename().c_str(), static_cast<int>(scene->GetManifestFilename().size()));
```

Comment:
is it possible for the scene to not have a manifest filename?

Reply:
I don't think so, the string is generated from the file name of the FBX/STL/GTLF file.